packages/web/src/services/solana/WithdrawUSDC.ts

```diff
@@ -46,43 +45,43 @@ export const getFundDestinationTokenAccountFees = async (
  * Creates instructions to swap from a user bank token into the given wallet as SOL.
  * These instructions are allowed in relay because every created token account is closed in the same transaction.
  */
-export const createSwapUserbankToSolInstructions = async ({
-  mint,
+export const swapUserBankUSDCToSol = async ({
```

Comment:
I updated this function to use SDK, but then ultimately moved away from using it. It's dead code. I figure I'll merge this, then remove in a subsequent PR so we have a piece of history that has an updated version of this swap in it.

Reply:
nit: Might be worth adding a deprecation docstring to it?